README.md

```diff
@@ -80,6 +90,24 @@ var app = express();
 
 // Your code starts here...
 
+get post put delete
+
+app.get("/bands", function(req, res){
+  res.send("index")
+})
+
+app.post("/bands", function(req, res){
+  res.send("new")
+})
+
+app.put("/:name", function(req, res){
```

Comment:
👍  url would probably would be `/bands/:name` following your example for both `Update` and `Delete`